src/data/migrations/CNew-03-request.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+CREATE TABLE IF NOT EXISTS "testers" (
+    "testerId" SERIAL,
+    "email" VARCHAR NOT NULL,
```

Comment:
Can we make email the primary key?

Reply:
Id as PK and make email unique.

Reply:
The email is unique, and the user ID is the PK. :)